### ISSUE TITLE
Exhaustive inference

### ISSUE THREAD

**Issue by o1lo01ol1o** [acowley/Frames#147]
I've run into some cases where I'd willingly trade the execution time of some long running computation for a single set of inferred types that I could manually include in source.  This clearly wouldn't work with the template haskell as it currently stands -- where could I splice such a function?  

In my mind, the expected behavior would be like `tableTypes` but would check more (all) rows and return a string representing all the generated code.

**Comment by acowley** [comment]
Yeah, I want this, too. There are tools for expanding TH splices into concrete text, but I think they’ve always been a bit fragile. Perhaps we should make a helper executable for `Frames` that doesn’t do TH at all, but just generates the right Haskell source, perhaps with a token in a comment so it can be updated in-place.
